build.gradle

```diff
@@ -123,21 +123,25 @@ subprojects { project ->
   checkerFramework {
     checkers = [
              'com.amazon.checkerframework.compliance.kms.ComplianceChecker',
-             'com.amazon.checkerframework.cryptopolicy.CryptoPolicyComplianceChecker'
+             'com.amazon.checkerframework.cryptopolicy.CryptoPolicyComplianceChecker',
+             'org.checkerframework.checker.https.HTTPSChecker'
     ]
-    extraJavacArgs = ['-Xmaxerrs', '1000', '-AsuppressWarnings=cast.unsafe', "-Astubs=$rootProject.projectDir/stubs"]
+    extraJavacArgs = ['-AprintVerboseGenerics', '-Xmaxerrs', '1000', '-AsuppressWarnings=cast.unsafe', "-Astubs=$rootProject.projectDir/stubs"]
   }
 
 //  apply plugin: 'net.ltgt.errorprone'
   dependencies {
 //    errorproneJavac 'com.google.errorprone:javac:9+181-r4173-1'
 //    errorprone 'com.google.errorprone:error_prone_core:2.3.3'
-    
-    checkerFramework 'com.amazonaws:kms-compliance-checker:1.0'
-    checkerFramework 'com.amazonaws:aws-crypto-compliance-checker:1.0'
-    implementation 'com.amazonaws:aws-crypto-compliance-checker:1.0'  
+
+    checkerFramework 'software.amazon.checkerframework:aws-kms-compliance-checker:1.0.2'
+    implementation 'software.amazon.checkerframework:aws-kms-compliance-checker:1.0.2'
```

Comment:
You shouldn't need this, since the kms checker doesn't provide any annotations of its own.